bin/e2e

```diff
@@ -16,12 +16,17 @@ fi
 
 printf "\n"
 
-read -p "Do you need to set up your end to end (E2E) testing database? Answer yes if this is your first time running E2E tests on your local machine or you need to recreate your E2E test database. (y/n) " -n 1 -r
+# Wait 10 seconds and if the user does not respond, assume they do not want to rebuild the test E2E database
+read -t 10 -p "Do you need to set up your end to end (E2E) testing database? Answer yes if this is your first time running E2E tests on your local machine or you need to recreate your E2E test database. (y/n) " -n 1 -r
```

Comment:
should we tell the user about about the timeout?

```suggestion
read -t 10 -p "Choose yes to setup a test database or if you're running the script for the first time, choose no or wait 10 seconds otherwise (y/n) " -n 1 -r
```